src/_gettsim/sozialversicherung/rente/grundrente/wartezeit.yaml

```
@@ -0,0 +1,21 @@
+---
+berücksichtigte_wartezeit:
```

```suggestion
berücksichtigte_wartezeit_monate:
```
We are using that for the column names these are compared to. Also,  `wartezeit` is measured in years in the more common contexts, so I think it is helpful to be explicit here.